ReactWindows/ReactNative/ReactPage.cs

```diff
@@ -193,7 +195,7 @@ protected virtual ReactRootView CreateRootView()
                 }
                 else if (e.EventType == CoreAcceleratorKeyEventType.KeyUp && _isControlKeyDown && e.VirtualKey == VirtualKey.R)
                 {
-                    await _reactInstanceManager.DevSupportManager.CreateReactContextFromPackagerAsync(CancellationToken.None).ConfigureAwait(false);
+                   _reactInstanceManager.DevSupportManager.HandleReloadJavaScript();
```

Comment:
>HandleReloadJavaScript(); [](start = 59, length = 25)

I thought I was simplifying things by removing an API - but I guess it's silly since almost all callers would be void returning.

Reply:
The reason I added this was different. I needed an "entrypoint" that acquired that awaitable async lock. CreateReactContextFromPackagerAsync is not one of those, it assumes the lock is taken. And I didn't kill my brain to try to make that lock allow recursion (not even sure how I would define that :) ).
It's a little bit hard to follow since execution bounces between instance manager and that dev helper, but eventually it was fortunate I didn't have to make too many tweaks.

---
In reply to: [165955662](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r165955662) [](ancestors = 165955662)